src/controllers/item.controller.ts

```diff
@@ -25,35 +33,50 @@ const createItem = async (req: Request, res: Response): Promise<void> => {
 
 /**
  * Retrieves all items from the database.
- * @param _req - The request object.
+ * @param req - The request object.
  * @param res - The response object.
  * @returns Promise that resolves to void.
  */
-const getItems = async (_req: Request, res: Response): Promise<void> => {
+const getItems = async (req: Request, res: Response): Promise<void> => {
+    const { page } = req.params;
+    const {name, category} = req.query;
```

Comment:
Es **req.params** o **req.query**?

Los parámetros son los que van después del "?" `/items?category=FRUITS&page=3`, creo que con ese **req.params** iría así `/items/3?category=FRUITS`. Quizás baste con mover esta variable a la línea 42.
```suggestion
    const { name, category, page } = req.query;
```